content/docs/user-guide/how-to/undo-dvc-add.md

```diff
@@ -0,0 +1,47 @@
+# Undo dvc add
```

Comment:
Not sure about this title. It has a command name in it but no back quotes, and we don't want to use `` ` `` here either, it will look strange...

Reply:
Can you come up with some alternatives that can be capitalized like a normal H1 header? Thanks

Reply:
Changed it to `Undo adding data`.

Reply:
OK. That or "Undo add Command" could work. Let's keep as-is for now.